lib/compiler/test/compile_SUITE.erl

```diff
@@ -1803,6 +1805,29 @@ deterministic_paths_1(DataDir, Name, Opts) ->
         file:set_cwd(Cwd)
     end.
 
+%% The test case uses ssh.erl from ssh application.
+deterministic_docs(Config) when is_list(Config) ->
+    DataDir = proplists:get_value(data_dir, Config),
+    Filepath = filename:join(DataDir, "ssh"),
+    false = deterministic_docs_1(Filepath, [binary], 100),
```

Comment:
Are 100 iterations needed to reliably trigger the fault? Or can it be reduced to limit runtime of the test?

Reply:
> Are 100 iterations needed to reliably trigger the fault? Or can it be reduced to limit runtime of the test?

Sure, number of iterations can be reduced, but chances of the first case (which is without 'deterministic' option) failure will increase.

(after some testing)
With 10 iterations is fails sometimes.
It seems, optimal number of is 25, no failures at least on my local machine.

Reply:
> It seems, optimal number of is 25, no failures at least on my local machine.

Changed number of iterations to 25.

Reply:
Great, thanks!